build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
 	implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
 	implementation("org.jetbrains.kotlin:kotlin-reflect")
 	implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+	compile("org.springframework.boot:spring-boot-starter-data-mongodb")
```

Comment:
Non-blocker: Consider not using the Spring Boot starter dependency, but rather the full on `spring-data-mongodb` (https://mvnrepository.com/artifact/org.springframework.data/spring-data-mongodb/2.1.8.RELEASE).